common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
The metadata shouldn't cross on the control plane. In fact I expect we can also do what @jthomas is alluding to. In general we have a deficiency today where a client may have an activation id but that id is not recognied by the system until the activation is completed and recorded by the invoker. 

This is wrong and we'll need to address it. Which means recording potentially similar metadata (say in redis for example) so that an query on the id will inform the client when the request was accepted, where it is in the processing pipeline, etc. 

To do that I expect the same metadata used for this tracing can do double duty.